percona/controller/pgcluster/controller.go

```diff
@@ -160,5 +165,60 @@ func (r *PGClusterReconciler) getVersionMeta(ctx context.Context, cr *v2beta1.Pe
 		PGVersion:       strconv.Itoa(cr.Spec.PostgresVersion),
 		BackupVersion:   "",
 		PMMVersion:      "",
-	}, nil
+		PMMEnabled:      cr.Spec.PMM != nil && cr.Spec.PMM.Enabled,
+	}
+	if _, ok := cr.Labels["helm.sh/chart"]; ok {
+		vm.HelmDeployCR = true
+	}
+	for _, set := range cr.Spec.InstanceSets {
+		if len(set.Sidecars) > 0 {
+			vm.SidecarsUsed = true
+			break
+		}
+	}
+	operatorDepl, err := r.getOperatorDeployment(ctx)
+	if err != nil {
+		return version.Meta{}, errors.Wrap(err, "failed to get operator deployment")
+	}
+	if _, ok := operatorDepl.Labels["helm.sh/chart"]; ok {
+		vm.HelmDeployOperator = true
+	}
+	return vm, nil
+}
+
+func (r *PGClusterReconciler) getOperatorDeployment(ctx context.Context) (*appsv1.Deployment, error) {
```

Comment:
@pooknull this method causes tests to fail, pls check it out. I think you should create operator deployment by hand in the test so this can go through.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/395/commits/5bd745d818a72ce1569065aa6b40b80802d4100e